aula-css/exercicios/exercicio 1 css/detalhes-1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="details.css">
+</head>
+<header>
+    <div class="titulo">
+        
+        <div>
+            <p><h1>Leila´s Photo Grid</h1></p>
+            
+        </div>
+        <br>
+        <div class="botao">
+        
+            <a href="index.html"><button> < Voltar </button> </a>
```

Comment:
```suggestion
            <a href="index.html" class="button">&lt; Voltar </a>
```
Motivo da sugestão: como a  interface não sugere um botão e sim um link, não há motivo para usar o elemento `<button>`, reserve-o para situações onde a interface deverá apresentar de fato um botão, como por exemplo em formulários (botões de enviar, cancelar, salvar). 
O uso do `&lt;` ao invés de < é para garantir que esse símbolo será interpretado como um texto e não como código.